Index: app/src/main/res/layout/activity_register.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_register.xml	(revision 31f6a5b50886f9c757e934368de06ee6bb7381db)
+++ app/src/main/res/layout/activity_register.xml	(date 1550689974867)
@@ -120,9 +120,23 @@
         style="?android:attr/progressBarStyle"
         android:layout_width="wrap_content"
         android:layout_height="45dp"
-        app:layout_constraintBottom_toBottomOf="@+id/btnRegister"
-        app:layout_constraintEnd_toEndOf="@+id/btnRegister"
-        app:layout_constraintStart_toStartOf="@+id/btnRegister"
-        app:layout_constraintTop_toTopOf="@+id/btnRegister" />
+        tools:layout_editor_absoluteX="182dp"
+        tools:layout_editor_absoluteY="531dp" />
+
+    <Button
+        android:textColor="#ffffff"
+        android:id="@+id/btnBackLogin"
+        android:layout_width="121dp"
+        android:layout_height="75dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginTop="26dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:background="@drawable/reg_btn_style"
+        android:text="Back"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/loadingProgress" />
 
 </android.support.constraint.ConstraintLayout>
\ No newline at end of file
Index: app/src/main/java/com/example/convoy/AccountActivity/RegisterActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/convoy/AccountActivity/RegisterActivity.java	(revision 31f6a5b50886f9c757e934368de06ee6bb7381db)
+++ app/src/main/java/com/example/convoy/AccountActivity/RegisterActivity.java	(date 1550689974858)
@@ -45,6 +45,7 @@
     private EditText regName, regMail, regPassword, confPassword;
     private ProgressBar loadingProgress;
     private Button btnRegister;
+    private Button btnBack;
 
     private FirebaseAuth mAuth;
 
@@ -57,15 +58,24 @@
         regMail = findViewById(R.id.regMail);
         regPassword = findViewById(R.id.regPassword);
         confPassword = findViewById(R.id.confPassword);
+        btnBack = findViewById(R.id.btnBackLogin);
         regName = findViewById(R.id.regName);
         loadingProgress = findViewById(R.id.loadingProgress);
         btnRegister = findViewById(R.id.btnRegister);
         loadingProgress.setVisibility(View.INVISIBLE);
 
 
+
         mAuth = FirebaseAuth.getInstance();
 
-
+        btnBack.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent backLogin = new Intent(getApplicationContext(),LoginActivity.class);
+                startActivity(backLogin);
+                finish();
+            }
+        });
         btnRegister.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
